何もしてないのに壊れた

多分依存パッケージが原因
- [ビルド成功](https://github.com/opthub-org/pytorch-bsf/actions/runs/23409484355/job/68094085381)
- [ビルド失敗](https://github.com/opthub-org/pytorch-bsf/actions/runs/23422091637/job/68129259367)

パッケージの差分
- starlette [0.52.1](https://github.com/opthub-org/pytorch-bsf/actions/runs/23409484355/job/68094085381#step:8:4652) -> [1.0.0](https://github.com/opthub-org/pytorch-bsf/actions/runs/23422091637/job/68129259367#step:8:4634)

エラー
```
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/mlflow/pyfunc/scoring_server/__init__.py", line 511, in init
    @app.route("/ping", methods=["GET"])
     ^^^^^^^^^
AttributeError: 'FastAPI' object has no attribute 'route'. Did you mean: 'router'?
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/test-env/bin/mlflow", line 6, in <module>
    sys.exit(cli())
             ~~~^^
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/click/core.py", line 1485, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/click/core.py", line 1406, in main
    rv = self.invoke(ctx)
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/click/core.py", line 1873, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/click/core.py", line 1873, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/click/core.py", line 1269, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/click/core.py", line 824, in invoke
    return callback(*args, **kwargs)
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/mlflow/models/cli.py", line 104, in serve
    ).serve(
      ~~~~~^
        model_uri=model_uri, port=port, host=host, timeout=timeout, enable_mlserver=enable_mlserver
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/usr/share/miniconda/envs/test-env/lib/python3.14/site-packages/mlflow/pyfunc/backend.py", line 328, in serve
    raise Exception(
        f"Command '{command}' returned non zero return code. Return code = {rc}"
    )
Exception: Command '['bash', '-c', 'exec uvicorn --host localhost --port 5001 --workers 1 mlflow.pyfunc.scoring_server.app:app']' returned non zero return code. Return code = 1
examples/quickstart/run.sh: line 1: kill: (3146) - No such process
Server process (PID 3146) has died unexpectedly.
Error: Process completed with exit code 1.
```

--- a/environment.yml
+++ b/environment.yml
@@ -10,7 +10,7 @@ dependencies:
 - lightning
 - torchmetrics
 - mlflow
-- starlette < 1.0.0  # hotfix for mlflow's starlette 1.0.0 issue
+- starlette<1.0.0  # hotfix for mlflow's starlette 1.0.0 issue
 - jsonschema
 - pyyaml
 - pip

--- a/setup.cfg
+++ b/setup.cfg
@@ -45,7 +45,8 @@ install_requires =
   pl-crossvalidate
   protobuf
   mlflow
-  starlette < 1.0.0  # hotfix for mlflow's starlette 1.0.0 issue
+  # hotfix for mlflow's starlette 1.0.0 issue
+  starlette<1.0.0
   jsonschema
   PyYAML
 

